Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/internal/Isotropic_remeshing/remesh_impl.h

```diff
@@ -199,14 +200,15 @@ namespace internal {
                                       PMP::parameters::edge_is_constrained_map(ecmap)
                                      .face_index_map(fimap));
         if(nb_cc == 1){
-          CGAL::internal::remove_property(patch_ids_map, pmesh);
+          // CGAL::remove_property(patch_ids_map, pmesh);
+          patch_ids_map = Patch_ids_map();
```

Comment:
I don't see anywhere that a property is default constructible. I guess, you want to call get another time.

Reply:
@sloriot   So I have to document that it is default constructible?

Reply:
yes

Reply:
As we do not have a manual page for `boost::property_map<CGAL::dynamic_vertex_property<T>>::type`  we can only state it informally in the User Manual section. Do you agree?